testcase/regression/peeloffbusy/test.sh

```diff
@@ -0,0 +1,25 @@
+Name="peeloffbusy"
+
+do_setup()
+{
+	sctp_darn -H 127.0.0.1 -P 5001 -l > /dev/null 2>&1 &
+	gcc client.c -o client -lsctp -lpthread
+}
+
+do_clean()
+{
+	local logf="$(st_o).log"
+
+	pkill sctp_darn
```

Comment:
Here I think you can avoid the pkill call by saving the sctp_darn pid above in a global variable
global pid
pid=$!  should be enough